workbench/tests/jest.overrides.js

```diff
@@ -23,6 +25,25 @@ if (global.window) {
   global.window.fetch = fetch;
 }
 
+mockI18n
+  .use(initReactI18next)
+  .init({
+    lng: 'en',
+    interpolation: {
+      escapeValue: false,
+    },
+    resources: {
+      ll: {
+        translation: {
+          Open: 'σρєи',
+          Language: 'ℓαиgυαgє'
+        }
+      }
+    },
+  });
+
+jest.mock('../src/shared/i18n', () => mockI18n);
```

Comment:
Cool, if this is moved to `main` and loaded via preload, I think there will be a chance to mock it right in this file around line 16.